src/nl/hannahsten/texifyidea/editor/LatexQuoteInsertHandler.kt

```diff
@@ -77,6 +83,12 @@ open class LatexQuoteInsertHandler : TypedHandlerDelegate() {
             if (previousChar == "{" || previousChar == "[" || previousChar == "(") {
                 isOpeningQuotes = true
             }
+
+            // If we are not closing the command, assume we are opening it (instead of doing nothing)
```

Comment:
Typo: _command_ to _quote_ (?)

Reply:
Artificially fixed in 64435398e86adaed89202a7a9c821025028b1d01